### PR TITLE
Fix: missing 'mbExpandFileName' invocations for 273759d1 for issue #130

### PR DESCRIPTION
**Pull request by TheNicker**

### CODE REVIEW

**Comment by TheNicker** [comment]
Apparently we forgot to invoke  'mbExpandFileName'. 
Use cases from #130 seems to work now

File: src/fileviews/ufileviewheader.pas [hunk 1]
```diff
@@ -234,8 +234,8 @@ procedure TFileViewHeader.onKeyRETURN(Sender: TObject);
   NewPath: String;
   AClass: TFileSourceClass;
 begin
-  NewPath:= NormalizePathDelimiters(FPathEdit.Text);
   NewPath:= ReplaceEnvVars(ReplaceTilde(NewPath));
```
Comment (alexx2000) [inline src/fileviews/ufileviewheader.pas:237]: `FPathEdit.Text` and `NewPath` are confused here.

Comment (TheNicker) [reply src/fileviews/ufileviewheader.pas:237]: It's on purpose. 
it's the same as in [fmain.pas:5903](https://github.com/doublecmd/doublecmd/pull/1259/files#diff-7b416c83e34c76fdacbb6e2854de3c561dd8177c9401896c5e8df0677671d1cbR5903) and [fmain.pas:5910](https://github.com/doublecmd/doublecmd/pull/1259/files#diff-7b416c83e34c76fdacbb6e2854de3c561dd8177c9401896c5e8df0677671d1cbR5910)

Shouldn't it be the same for both code paths ?

Comment (alexx2000) [reply src/fileviews/ufileviewheader.pas:237]: Я не про порядок вызова функций, сейчас выходит такой код:
```
var
  NewPath: String;
  AClass: TFileSourceClass;
begin
  NewPath:= ReplaceEnvVars(ReplaceTilde(NewPath));
  NewPath:= mbExpandFileName(Trim(FPathEdit.Text));
```
`NewPath` в первой строке пустой. Там должен быть `Trim(FPathEdit.Text)`, а уже во второй `NewPath`.

Comment (TheNicker) [reply src/fileviews/ufileviewheader.pas]: Fixed

**Pull request merged**